dll/win32/shell32/CDefView.cpp

```diff
@@ -71,6 +80,25 @@ typedef struct
 // to call TrackPopupMenu and let it use the 0 value as an indication that the menu was canceled
 #define CONTEXT_MENU_BASE_ID 1
 
+struct PERSISTCOLUMNS
```

Comment:
Could you explain why we need signatures for these structs? Apologies if this is a stupid question.

Reply:
Need is a strong word. We are reading binary data from the registry here and in this case, just an array of UINTs. Without a signature, I have no way of verifying that this data is at least somewhat correct.

Failing to read the data just means we forget to restore the previously configures columns in this folder and just end up with the default column configuration instead,

Reply:
Good enough for me, though I do think when we get a transactional registry this will be unnecessary.